crates/sui-faucet/src/faucet/simple_faucet.rs

```diff
@@ -45,6 +46,8 @@ pub struct SimpleFaucet {
     consumer: Mutex<Receiver<ObjectID>>,
     pub metrics: FaucetMetrics,
     wal: Mutex<WriteAheadLog>,
+    request_producer: Mutex<Sender<(Uuid, SuiAddress, Vec<u64>)>>,
+    request_consumer: Mutex<Receiver<(Uuid, SuiAddress, Vec<u64>)>>,
```

Comment:
I believe the mutexes should be unnecessary here.  On the consumer/rx side, you should only have one consumer flushing the queue, and on the producer side, `Sender` [implements `Sync`](https://docs.rs/tokio/latest/tokio/sync/mpsc/struct.Sender.html#impl-Sync-for-Sender%3CT%3E) meaning it is safe to send references to `Sender` across threads and not suffer races.